src/joystick/linux/SDL_sysjoystick.c

```diff
@@ -1259,7 +1483,13 @@ static int LINUX_JoystickSendEffect(SDL_Joystick *joystick, const void *data, in
 
 static int LINUX_JoystickSetSensorsEnabled(SDL_Joystick *joystick, SDL_bool enabled)
 {
-    return SDL_Unsupported();
+    if (!(joystick->hwdata->has_accelerometer || joystick->hwdata->has_gyro)) {
```

Comment:
Please change this to:
```c
if (!joystick->hwdata->has_accelerometer && !joystick->hwdata->has_gyro) {
```
In general, SDL code tries to be easy to read aloud.

Reply:
I prefer this way because it is easier to see the opposite condition that go with the "else", but I'll change it.